docs/contributors/guide/cherry-pick/cherry-pick.md

```diff
@@ -0,0 +1,123 @@
+- Memory corruption
+- Panic, crash, hang
+- Security
+- Tests、Metrics、Logging
+
+A bugfix for a functional issue (not a data loss or security issue) does not qualify as a critical bug fix.
+
+If you are proposing a cherry pick and it is not a clear and obvious critical bug fix, please reconsider. If upon
+reflection you wish to continue, bolster your case by supplementing your PR with e.g.,
+
+- A GitHub issue detailing the problem
+
+- Scope of the change
+
+- Risks of adding a change
+
+- Risks of associated regression
+
+- Testing performed, test cases added
+
+## Initiate a Cherry Pick
+
+- Add `needs-cherry-pick-release-{version}` label to the PR, the robot will then automatically create a new PR for the
+  cherry pick.
+- Your cherry pick PR will immediately get the `do-not-merge/cherry-pick-not-approved` label.
+
+## Cherry Pick Review
+
+Unlike other PRs, cherry pick PR will directly copy the LGTMs of the original PR, but we still require `/merge` to
+trigger the merge PR.
+
+The same release note requirements apply as normal pull requests, except the release note stanza will auto-populate from
+the master branch pull request from which the cherry pick originated.
+
+If this is unsuccessful, the `do-not-merge/release-note-label-needed` label will be applied and the cherry pick author
+must edit the pull request description to add a release note or include in a comment the `/release-note-none` command.
+
+Cherry pick pull requests are reviewed slightly differently than normal pull requests on the `master` branch in that
+they:
+
+- Are by default expected to be `type/bug-fix`.
+
+- The original change to the `master` branch is expected to be merged for some time and no related CI failures or test
+  flakiness must be discovered.
+
+- Milestones must be set on the PR reflecting the milestone for the target release branch (for example, milestone v5.3.2
+  for a cherry pick onto branch
+  `release-5.3`). This is normally done for you by manually.
+
+- A separate cherry pick pull request should be open for every applicable target branch. This ensures that the fix will
+  be present on every active branch for a given set of patch releases. If a fix is only applicable to a subset of active
+  branches, it is helpful to note why that is the case on the parent pull request or on the cherry pick pull requests to
+  the applicable branches.
+
+- Have one additional level of review in that they must be approved specifically for cherry pick by branch approvers.
+
+  Approval is signified by an approver manually applying the
+  `cherry-pick-approved` label. This action removes the
+  `do-not-merge/cherry-pick-not-approved` label and triggers a merge into the target branch.
+
+  If you are concerned about the status of your cherry pick, err on the side of overcommunicating and reach out to the
+  [nongfushanquan](https://github.com/nongfushanquan). We should contact [nongfushanquan](https://github.com/nongfushanquan) as early as possible for approval.
```

Comment:
Is there any github team that can approve cherry-picks?

Reply:
No